### ISSUE TITLE
`gwdt`: corresponds roughly to the builtin `graydist`, but has a different interface.

### ISSUE THREAD

**Issue by wkearn** [TopoToolbox/topotoolbox3#22]

**Comment by wkearn** [comment]
Once #45 is merged, this will be the final component of #15. The binding of libtopotoolbox's `gwdt` function should not be too complicated: the instructions in #45 should be sufficient to create a MEX file that does the right thing. 

The challenge is that `gwdt` expects to receive the 32-bit integer encoded `flats` array that is created by libtopotoolbox's `identifyflats`. This labels the flats, sills and presills in the same array, and we don't currently return it from topotoolbox3's `identifyflats`. Instead, we split it up into three boolean arrays for flats, sills and closed basins, and then use the `getPresillPixels` helper function in `createAuxiliaryTopo` to label the presills.

It might be easiest if, when using libtopotoolbox, the MEX function `iflats = tt_identifyflats(single(DEM))` is used directly, and this int32 `iflats` array is passed around instead of `I`, `SILLS`, etc. `tt_gwdt_computecosts`, for example, can use `iflats` instead of `int32(I)` without any other changes.
